Vagrantfile

```diff
@@ -30,8 +30,8 @@ sudo apt-get update -qq -y
 
 sudo apt-get install -qq curl php5 git libspreadsheet-writeexcel-perl libdbd-sqlite3-perl
 
-curl -sS https://getcomposer.org/installer | php
-sudo mv composer.phar /usr/bin/composer
+make composer_download
+sudo make composer_install
```

Comment:
The `composer_install` target is also called later when `sudo make install` is called (see: https://github.com/fossology/fossology/blob/ae754b26e849b4c9068ed6f4aa354310165aec0c/src/Makefile#L53)